app/packages/core/src/plugins/SchemaIO/components/SliderView.tsx

```
@@ -1,46 +1,274 @@
+    if (!value) {
+      isMin ? setMinText("") : setMaxText("");
+    } else if (unit === "%") {
+      const percentageValue = parseInt(value);
+      if (percentageValue >= 0 && percentageValue <= 100) {
+        isMin
+          ? setMinText(percentageValue.toFixed())
+          : setMaxText(percentageValue.toFixed());
+      }
+    } else {
+      const floatValue = parseFloat(value);
+      if (!isNaN(floatValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)